src/osd/OSD.cc

```diff
@@ -1782,20 +1782,20 @@ class OSDSocketHook : public AdminSocketHook {
   OSD *osd;
 public:
   explicit OSDSocketHook(OSD *o) : osd(o) {}
-  bool call(std::string command, cmdmap_t& cmdmap, std::string format,
+  bool call(std::string admin_command, cmdmap_t& cmdmap, std::string format,
```

Comment:
i don't understand how this helps us to search the admin_command? IIUC, all commands in OSD.cc are admin commands.